internal/elasticsearch/common.go

```diff
@@ -610,22 +627,12 @@ func newVolumeSource(clusterName, nodeName, namespace string, node api.Elasticse
 		StorageClassName: specVol.StorageClassName,
 	}
 
-	err := createOrUpdatePersistentVolumeClaim(volSpec, claimName, namespace, clusterName, client)
+	_, err := persistentvolume.CreateOrUpdatePVC(context.TODO(), client, pvc, persistentvolume.LabelsEqual, persistentvolume.MutateLabelsOnly)
```

Comment:
why return an object if we don't use it? can we update this to just return an error?

Reply:
Done